src/ResourceManager/PowerBIEmbedded/Commands.PowerBIEmbedded.Test/ScenarioTests/Common.ps1

```diff
@@ -0,0 +1,110 @@
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Creates a unique name for a resource group
+#>
+function Get-ResourceGroupName
+{
+	return "pstest" + $(Get-Random) + "RG"
+}
+
+<#
+.SYNOPSIS
+Creates a unique name for a workspace collection
+#>
+function Get-WorkspaceCollectionName 
+{
+	return "pstest" + $(Get-Random) + "WC"
+}
+
+<#
+.SYNOPSIS
+Cleans the created resource groups
+#>
+function Clean-ResourceGroup($rgname)
+{
+    if ([Microsoft.Azure.Test.HttpRecorder.HttpMockServer]::Mode -ne [Microsoft.Azure.Test.HttpRecorder.HttpRecorderMode]::Playback) {
+        Remove-AzureRmResourceGroup -Name $rgname -Force
+    }
+}
+
+<#
+.SYNOPSIS
+Creates a new resource group
+#>
+function Create-ResourceGroup
+{
+	$resourceGroupName = Get-ResourceGroupName
+	return New-AzureRmResourceGroup -Name $resourceGroupName -Location WestUS
```

Comment:
Don't hardcode the location, as this might not be available dpending on what evironment or what subscription is used to run a test.  You can use a simple function to get the available provider locatiosn and prefer a location liek West US if it is available.